LabDrivers/Tool.py

```diff
@@ -535,6 +535,29 @@ def change_interface(self, intf, **kwargs): # only really applicable to prologix
         else:
             logging.debug("Change interface called with %s, current interface is %s"%(intf, self.interface))
 
+
+    def set_ren(self, level):
+        # this function is used as an alias for control_ren, which is built in to pyvisa controller, and programmed into Prologix
```

Comment:
Use preferably docstrings, rather than a comment. A docstring will be displayed upon typing `help(set_ren)`. See https://www.python.org/dev/peps/pep-0257/ for best practices :)